packages/webview_flutter/webview_flutter_android/lib/src/instance_manager.dart

```diff
@@ -23,51 +23,179 @@ mixin Copyable {
   Copyable copy();
```

Comment:
Copy of: https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter_wkwebview/lib/src/common/instance_manager.dart